EIPS/eip-4844.md

```diff
@@ -166,7 +166,7 @@ The signature is verified and `tx.origin` is calculated as follows:
 ```python
 def tx_hash(tx: SignedBlobTransaction) -> Bytes32:
```

Comment:
Change looks good, and don't mean to bike-shed, but I think we should clarify the function naming while we're at it.

We should clarify that this outputs the hash to sign & use for verification, not the full tx hash, which would be easy to mistake when it's called `tx_hash`.

And the other method, "signed_tx_hash", is confusing with CL specs context (we use "signing_root" to describe the hash that is used for signing, and thus doesn't include the signature values yet).

Maybe we can be explicit and say `unsigned_tx_hash` (here) and `signed_tx_hash` (the other PR, already approved)?

Reply:
Agree .. will clarify.

Reply:
@protolambda what about "signing_root_hash" for this one and "tx_id_hash" for the other?

Reply:
it's not a root since it's not a tree commitment anymore, and it's not really an ID either (many transaction managers use "ID" to track transactions that may be resubmitted with different fee parameters and thus different hash). I'd just keep it simple and do `{signed, unsigned}_tx_hash`

Reply:
Would also be fine with `signing_hash` and `tx_hash` if you prefer

Reply:
changed to {signed,unsigned}_tx_hash